auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -799,6 +811,20 @@ protected boolean isReadRight(String action) {
         return false;
     }
 
+    private List<Boolean> hasRightsBasedOnSingleCheckRightRequests(String memberCrn, List<RightCheck> rightChecks, AuthorizationClient client,
+            String requestId) {
+        return rightChecks.stream()
+                .map(check -> {
+                    try {
+                        client.checkRight(requestId, memberCrn, check.getRight(), check.getResource());
+                        return true;
+                    } catch (Exception ex) {
+                        return false;
```

Comment:
`makeCheckRightCall` has the same exception driven return statement, is it possible to reuse that?